epochStart/metachain/trigger.go

```diff
@@ -239,12 +245,17 @@ func (t *trigger) EpochStartMetaHdrHash() []byte {
 	return t.epochStartMetaHash
 }
 
+// GetSavedStateKey returns the last saved trigger state key
+func (t *trigger) GetSavedStateKey() []byte {
+	return t.triggerStateKey
+}
+
 // SetEpochStartMetaHdrHash sets the epoch start meta header hase
 func (t *trigger) SetEpochStartMetaHdrHash(metaHdrHash []byte) {
 	t.epochStartMetaHash = metaHdrHash
 }
 
-// SetLastEpochStartRound sets the round when the current epoch started
+// SetCurrentEpochStartRound sets the round when the current epoch started
 func (t *trigger) SetCurrentEpochStartRound(round uint64) {
 	t.mutTrigger.Lock()
 	t.currentRound = round
```

Comment:
I think here t.currEpochStartRound has to be set.

Reply:
there is a function in sync/metaBlock - line 205 - setEpochStart - it can be deleted now. not more needed.